tests/constants.ts

```diff
@@ -0,0 +1,6 @@
+// They are connected 
+export const PUBKEY_HEX = "04adf6cad45e58a7e1908bebefcc358de229c108fb1170566f83be5ce028eb6b1997711067ffcb445532667ed4a4efc2b334c16421edb52ec5e0324a1c0e570663"
+export const BECH32_ACCOUNT = "cheqd1ehcg0jarxkyxtkzrwcxayedxrskwyftxj4exm9"
+// Testnet RPC nad faucet address
+export const FAUCET_ADDRESS = "cheqd1rnr5jrt4exl0samwj0yegv99jeskl0hsxmcz96"
+export const TESTNET_RPC = "https://rpc.cheqd.network:443"
```

Comment:
Could you move all the above declarations under `src/tests/testutils.test.ts` instead?

I'd probably keep them all in lower case as well.

It's intended to hold all test utils, hence why it's overlooked by test config.

Reply:
Plus, faucet address is already defined there.